app/_kong_plugins/jwt-signer/index.md

```diff
@@ -153,3 +161,26 @@ reload tokens specified in [`config.access_token_jwks_uri`](/plugins/jwt-signer/
 rotate Keys **twice**, as it effectively replaces both current and previous Key Sets
 with newly generated tokens or reloaded tokens if the Keys were loaded from
 an external URI.
+
+## Claim validation {% new_in 3.12 %}
+
+In {{site.base_gateway}} 3.12 or later, you can perform additional claim validation by specifying the types of token claims as well as which tokens are required and which are optional. 
+
+You can validate the following types of access and channel token claims:
+* Issuer (`iss`)
+* Not before (`nbf`)
+* Audience (`aud`)
+* Subject (`sub`)
```

Comment:
```suggestion
* [Issuer](/plugins/jwt-signer/examples/validate-channel-token-issuers/) (`iss`)
* [Not before](/plugins/jwt-signer/examples/validate-access-token-issuers/) (`nbf`)
* [Subject](/plugins/jwt-signer/examples/validate-channel-token-subjects/) (`sub`)
* Audience (`aud`)
```
Moving `aud` to the bottom since we don't have an example for it, so it looks less awkward without a link.